crates/apub/src/objects/comment.rs

```diff
@@ -169,6 +170,7 @@ impl Object for ApubComment {
     let slur_regex = &local_site_opt_to_slur_regex(&local_site);
     let url_blocklist = get_url_blocklist(context).await?;
     let content = process_markdown(&content, slur_regex, &url_blocklist, context).await?;
+    let content = markdown_rewrite_remote_links(content, context).await;
```

Comment:
Unfortunately thats not possible because process_markdown is defined in lemmy_utils, but this function must be defined in lemmy_apub so that it can use activitypub functionality.